qutip/visualization.py

```diff
@@ -828,6 +948,11 @@ def matrix_histogram(M, x_basis=None, y_basis=None, limits=None,
                      options['bars_spacing'])
     ax.view_init(azim=options['azim'], elev=options['elev'])
 
+    # removing margins
+    _remove_margins(ax.xaxis)
+    _remove_margins(ax.yaxis)
+    _remove_margins(ax.zaxis)
```

Comment:
Does this need to be inside to loop so it's applied to all artists?

Reply:
No. This is already applied to all artists.